ouroboros-network/src/Ouroboros/Network/Diffusion.hs

```diff
@@ -816,7 +824,19 @@ runDataDiffusion tracers
                   connectionHandler
                   classifyHandleError
                   NotInResponderMode
-                  $ \(connectionManager :: NodeToNodeConnectionManager InitiatorMode Void) ->
+                  $ \(connectionManager :: NodeToNodeConnectionManager InitiatorMode Void) -> do
+#ifdef POSIX
+                  _ <- Signals.installHandler 
```

Comment:
Remove the space after `installHandler`.

Reply:
Done